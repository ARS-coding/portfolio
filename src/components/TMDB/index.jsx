import React from "react";
import { Col, Row } from "react-bootstrap";
import { VerticalTimelineElement } from "react-vertical-timeline-component";

const TMDB = () => {
  return (
    <VerticalTimelineElement
      className="vertical-timeline-element--work"
      date={"Sept 2020"}
      dateClassName={"date"}
      contentStyle={{
        background: "white",
        color: "white",
      }}
      contentArrowStyle={{
        borderRight: "7px solid  white",
      }}
      iconStyle={{
        background: "white",
      }}
      icon={<i className="fas fa-project-diagram projectIcon"></i>}
    >
      <div className="homeText projectContent">
        <h4>TMDB</h4>
        <small>Team Project</small> <br />
        <a
          href="https://github.com/tmansuroglu/TMDB-Project"
          target="_blank"
          rel="noreferrer"
        >
          Github
        </a>{" "}
        |{" "}
        <a
          href="https://tmdbproject.netlify.app/"
          target="_blank"
          rel="noreferrer"
        >
          Demo
        </a>
        <p className="projectText">
          <strong>Goal:</strong> creating an IMDB clone to learn object oriented
          programming.
        </p>
        <br />
        <Row>
          <Col xs={12} md={6}>
            <h5>Skills Applied</h5>
            <ul>
              <li>HTML</li>
              <li>CSS</li>
              <li>JavaScript</li>
              <li>Bootstrap</li>
              <li>Responsive Design</li>
              <li>API Usage</li>
            </ul>
          </Col>
          <Col xs={12} md={6}>
            <h5>My Responsibilities</h5>
            <ul>
              <li>Setting up TMDB API</li>
              <li>Movie/Tv Series Page</li>
              <li>Actor/Actress Page</li>
              <li>Search Page</li>
              <li>Main Page</li>
              <li>Navbar Dropdowns</li>
              <li>Visual Design</li>
            </ul>
          </Col>
        </Row>
      </div>
    </VerticalTimelineElement>
  );
};

export default TMDB;
