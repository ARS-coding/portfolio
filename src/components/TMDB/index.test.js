import React from 'react';
import renderer from 'react-test-renderer';
import TMDB from '.';

it('TMDB renders correctly', () => {
  const tree = renderer.create(<TMDB />).toJSON();
  expect(tree).toMatchSnapshot();
});
