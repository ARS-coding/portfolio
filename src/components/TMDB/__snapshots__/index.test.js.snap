// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TMDB renders correctly 1`] = `
<div
  className="vertical-timeline-element--work vertical-timeline-element"
  id=""
  style={null}
>
  <span
    className="vertical-timeline-element-icon is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
      }
    }
  >
    <i
      className="fas fa-project-diagram projectIcon"
    />
  </span>
  <div
    className="vertical-timeline-element-content is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
        "color": "white",
      }
    }
  >
    <div
      className="vertical-timeline-element-content-arrow"
      style={
        Object {
          "borderRight": "7px solid  white",
        }
      }
    />
    <div
      className="homeText projectContent"
    >
      <h4>
        TMDB
      </h4>
      <a
        href="https://github.com/tmansuroglu/TMDB-Project"
        rel="noreferrer"
        target="_blank"
      >
        Github
      </a>
       
      |
       
      <a
        href="https://tmdbproject.netlify.app/"
        rel="noreferrer"
        target="_blank"
      >
        Demo
      </a>
      <p>
        <strong>
          Goal:
        </strong>
         creating an IMDB clone.
      </p>
      <br />
      <div
        className="row"
      >
        <div
          className="col-md-6 col-12"
        >
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              Bootstrap
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              API Usage
            </li>
          </ul>
        </div>
        <div
          className="col-md-6 col-12"
        >
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Setting up TMDB API
            </li>
            <li>
              Movie/Tv Series Page
            </li>
            <li>
              Actor/Actress Page
            </li>
            <li>
              Search Page
            </li>
            <li>
              Main Page
            </li>
            <li>
              Navbar Dropdowns
            </li>
            <li>
              Visual Design
            </li>
          </ul>
        </div>
      </div>
    </div>
    <span
      className="date vertical-timeline-element-date"
    >
      Sept 2020
    </span>
  </div>
</div>
`;
